docs/header.md

```diff
@@ -17,8 +17,33 @@ The module has been tested with:
 - Materialize Operator v0.1.0
 
 ## Setup Notes:
-This module requires active azure credentials in your environment either setup through keys in the environment variable or through
-`az login` with azure's CLI.
 
-This module also runs an python script to generate Azure SAS tokens for the storage account. This requires python 3.12 or greater. Dependencies
-for the script can be found in the `requirements.txt` file.
+This module requires active Azure credentials in your environment, either set up through environment variables containing the required keys or by logging in with the Azure CLI using:
+
+```sh
+az login
+```
+
+Additionally, this module runs a Python script to generate Azure SAS tokens for the storage account. This requires **Python 3.12 or greater**.
+
+### Installing Dependencies
+
+Before running the module, ensure you have the necessary Python dependencies installed:
+
+1. Install Python 3.12+ if you haven't already.
+2. Install the required dependencies using `pip`:
+
+   ```sh
+   pip install -r requirements.txt
+   ```
+
+If you are using a virtual environment, you can set it up as follows:
+
+```sh
+python -m venv venv
+source venv/bin/activate  # On macOS/Linux
+venv\Scripts\activate  # On Windows
+pip install -r requirements.txt
+```
```

Comment:
just wondering -- should we symlink this requirements.txt file in the examples/simple directory as well?

Reply:
Thank you!  Running this now!  So excited!